### PR TITLE
38 import imd ruc data

### LINKED ISSUE #38: Import constituency-level IMD and RUC data
@ajparsons will be creating constituency-level equivalents of our local authority deprivation and rural/urban coefficient datasets:

* https://github.com/mysociety/composite_uk_imd/issues/3
* https://github.com/mysociety/uk_ruc/issues/1

Should be fairly simple to import into the platform once that’s done.

### PR DESCRIPTION
**Pull request by struan**
Fixes #38

### CODE REVIEW

File: hub/templates/hub/area.html [hunk 1]
```diff
@@ -195,8 +194,8 @@ <h5 class="card-title mb-3">2019 voter turnout</h5>
             <div class="col d-flex flex-column">
                 <div class="card flex-grow-1 fake-data">
```
Comment (zarino) [inline hub/templates/hub/area.html:195]: Left in a `fake-data` here.

File: hub/templates/hub/area.html [hunk 2]
```diff
@@ -183,9 +183,8 @@ <h5 class="card-title mb-3">Households in Fuel poverty</h5>
...
```
Comment (zarino) [inline hub/templates/hub/area.html:184]: Left in a `fake-data` here.

File: hub/templates/hub/area.html [hunk 3]
```diff
@@ -183,9 +183,8 @@ <h5 class="card-title mb-3">Households in Fuel poverty</h5>
...
                     <div class="card-body">
-                        <h5 class="card-title mb-3">2019 voter turnout</h5>
-                        <p class="card-text mb-0 display-6 lh-1 text-primary">65%</p>
-                        <p class="card-text mt-2 text-muted">59% national average</p>
+                        <h5 class="card-title mb-3">Index of multiple Deprivation</h5>
```
Comment (zarino) [inline hub/templates/hub/area.html:186]: Capital "M" for Multiple?

File: hub/templates/hub/area.html [hunk 4]
```diff
@@ -183,9 +183,8 @@ <h5 class="card-title mb-3">Households in Fuel poverty</h5>
...
+                        <p class="card-text mb-0 display-6 lh-1 text-primary">{{ constituency_imd.value }}</p>
                     </div>
                     <div class="card-footer bg-white">
                         <p class="card-text fs-8"><a href="https://commonslibrary.parliament.uk/general-election-2019-turnout/" class="text-decoration-none text-muted">Data from House of Commons Library</a></p>
```
Comment (zarino) [inline hub/templates/hub/area.html:190]: Source line needs to be updated.

File: hub/templates/hub/area.html [hunk 5]
```diff
@@ -195,8 +194,8 @@ <h5 class="card-title mb-3">2019 voter turnout</h5>
             <div class="col d-flex flex-column">
                 <div class="card flex-grow-1 fake-data">
                     <div class="card-body">
-                        <h5 class="card-title mb-3">‘Levelling up’ town</h5>
-                        <p class="card-text fs-5">Nearby <strong class="text-primary">Borchester</strong> received ‘levelling up’ funds in 2021</p>
+                        <h5 class="card-title mb-3">Urban Rural Classification</h5>
+                        <p class="card-text fs-5">{{ constituency_ruc.value }}</p>
                     </div>
                     <div class="card-footer bg-white">
                         <p class="card-text fs-8"><a href="#" class="text-decoration-none text-muted">Source: GOV.UK, 2021</a></p>
```
Comment (zarino) [inline hub/templates/hub/area.html:201]: Source line needs to be updated.

**Review by zarino — CHANGES_REQUESTED** [review]
Couple of comments inline.

~~Aside – I wonder whether the dataset source string (and link) should be passed to the template as dataset metadata, rather than being hard-coded? (Maybe you’ve already got this change in the works in another branch!)~~ — Indeed, now I see it’s already in https://github.com/mysociety/local-intelligence-hub/pull/58 :+1:

**Review by zarino — APPROVED** [review]

**Review by alexander-griffen — APPROVED** [review]

**Pull request merged**